definitions/control-v1.yaml

```diff
@@ -3801,6 +3807,256 @@ components:
       - requestMode
       - source
       - target    
+    pulsar_token_auth:
+      type: object
```

Comment:
Sorry... I just noticed that it's not picking up the type for `authentication` either(!)

Reply:
OK I will look into that! Thanks!